### PR TITLE
Added the event block to index.html

### PR DESCRIPTION
**Pull request by SachseInTheHub**

### CODE REVIEW

**Comment by arbarlow** [comment]
I think you've broken the YAML file?

File: source/stylesheets/_base.css.scss [hunk 1]
```diff
@@ -67,19 +67,23 @@ li {
 body {
   background-color: $background;
   min-width: 320px;
-  margin:0;
+  margin:2;
```
Comment (raffij) [inline source/stylesheets/_base.css.scss]: 2 what? px?

File: source/index.html.erb [hunk 2]
```diff
@@ -14,24 +14,18 @@
...
-	      	Brainstorming session with teachers to explore challenges and opportunities for teachers in the digital age. The goal was to learn about improving teacher's experience with technology in the classrooms and understand where the industry can help.
-	      </li>
-     	</ul>
-     	<div class="event-full-desc">
-     		Technology is nowhere close to replacing teachers but it is aiding the transformation of their roles. Teachers are excited about use of tech in their classroom. In order to implement it successfully, they need to understand how it fits with the curriculum, learning objectives, and the assessment system they are working with. We will be publishing more detailed learnings about designing for education soon.
-     	</div>
-  	</div>
-  	<% end %>
+	<% data.events.each do |event| %>
+		<div class="event-content">
```
Comment (raffij) [inline source/index.html.erb:18]: you sticking to 2px spaces for tabs?

File: source/stylesheets/_base.css.scss (hunk 1 above)
Comment (SachseInTheHub) [inline source/stylesheets/_base.css.scss]: @raffij em. good?

File: source/index.html.erb (hunk 2 above)
Comment (SachseInTheHub) [inline source/index.html.erb:18]: @arbarlow just helped me changed the sublime tabs width for 2!

**Pull request merged**